src/components/Menu.js

```diff
@@ -24,13 +29,6 @@ const Menu = ({ onMenuClick, logout }) => {
           sidebarIsOpen={open}
         />
       ))}
-      <MenuItemLink
-        to="/custom-route"
-        primaryText="Miscellaneous"
-        leftIcon={<LabelIcon />}
-        onClick={onMenuClick}
-        sidebarIsOpen={open}
-      />
```

Comment:
This seems to be never used and c&p issue.

 => https://marmelab.com/react-admin/doc/3.19/Admin.html#menu